src/CattyTests/DataModel/VariablesContainerTest.swift

```diff
@@ -459,6 +459,32 @@ final class VariablesContainerTest: XCTestCase {
         XCTAssertFalse(container.isProjectVariableOrList(list2))
     }
 
+    func testIsProjectVariable() {
+        let userVariable1 = UserVariable(name: "testVar1", isList: false)
+        let userVariable2 = UserVariable(name: "testVar2", isList: false)
+
+        let container = VariablesContainer()
+
+        container.programVariableList.add(userVariable1)
+        container.objectVariableList.setObject(userVariable1, forKey: "VariableKey")
```

Comment:
Please make use of the function `addObjectVariable` instead of adding them to the data structure directly.